getparty.go

```diff
@@ -424,6 +433,24 @@ func (cmd Cmd) applyHeaders(req *http.Request) {
 	}
 }
 
+func (cmd Cmd) getTransport() (*http.Transport, error) {
+	transport := cleanhttp.DefaultPooledTransport()
```

Comment:
`cleanhttp` provides two options `DefaultTransport` and`DefaultPooledTransport`.
Can you please add `pooled bool` param and based on it make either one.
Follow method used `cleanhttp.DefaultClient` which makes `DefaultTransport` under the hood.